src/locales/default.json

```diff
@@ -228,8 +228,14 @@
   },
   "setup": {
     "example": "e.g. yam.eth",
-    "useExistingEns": "Use an existing ENS name to create your space with.",
-    "createASpace": "Create a space"
+    "chooseExistingEns": "Chose one of your existing ENS domains to create a space with:",
+    "useSingleExistingEns": "Use your existing ENS domain:",
+    "orReigsterNewEns": "Or register a new domain:",
+    "toCreateASpace": "To create a space, you first need an ENS domain. Enter one below and follow the ENS registration instructions.",
+    "correctAccountNote": "(If you already own an ENS domain that you want to use, make sure you are connected to the correct account on the Ethereum mainnet.)",
```

Comment:
Not sure about the length of this. Maybe "Can't see your ENS address? Make sure you have connected the correct account on the Ethereum mainnet."

Reply:
Hmmm... Actually this was just supposed to cover the case where we don't find any domains for the connected account and thus show only the registration dialog. In that case, it might not be clear to the user that we actually do this lookup and that he/she is just connected to the wrong account. User might think: "Ah ok, so I need to input my domain here. But what you mean? Register? I already did that. Or... "Register" with Snapshot? Or what? (reads the note...) Ahhh sure... my fault. (switches account and domains appear)"

When there are domains, I believe it is clear enough to the user. I imagine having a few domains on this account and a few more on that one. If the UI shows me one of the sets, it would be pretty clear to me that I need to switch account if I want to use on of my other domains.

Reply:
> "You already have an ENS domain? Make sure you have connected the correct account on the Ethereum mainnet."

That's better yeah!

> And for the "some domains but not the ones I'm looking for" scenario, as explained, not sure if such a note is even necessary. If we decide to put it there too, then I like "Can't see your ENS address? ...". (I'd just write "domain" instead of "address".)

I don't think we need to cover this case.